tests/test_samplings.py

```diff
@@ -19,14 +19,11 @@ def test_cube_equidistant():
     np.testing.assert_allclose(x, coords.x)
     np.testing.assert_allclose(y, coords.y)
     np.testing.assert_allclose(z, coords.z)
+    assert isinstance(coords, Coordinates)
```

Comment:
```suggestion
    assert type(coords) is Coordinates
```
shoudnt we be more explict here? due to the SamplingSphere and Coordinates inheritance?

Reply:
Good point. I've replaced all `isinstances` with `type` to be future-proof